Sources/PT.PM.Common/Nodes/IAttributable.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace PT.PM.Common.Nodes
+{
+    public interface IAttributable
```

Comment:
Can we name it `IHaveAttributes` for clarity?

`IHazAttributes` would be even better, but I'm not insisting... :smile: